x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -415,7 +415,7 @@ export type PolicyInfo = Immutable<{
 
 export type HostInfo = Immutable<{
   metadata: HostMetadata;
-  host_status: HostStatus;
+  host_status?: HostStatus;
```

Comment:
I assume this was added because it's an enriched field and you are skipping agent enrichment?

Reply:
yes, it is from https://github.com/elastic/kibana/blob/181aa5f8b58244c152642300c92df6c5e22c0916/x-pack/plugins/security_solution/server/endpoint/routes/metadata/handlers.ts#L311